Resources/services/subscribers.xml

```diff
@@ -328,5 +328,13 @@
                  public="true">
             <tag name="shopware.event_subscriber"/>
         </service>
+
+        <service id="paypal_unified.subscriber.transaction_report"
+                 class="SwagPaymentPayPalUnified\Subscriber\TransactionReportSubscriber"
+                 public="true">
+            <argument type="service" id="dbal_connection"/>
+            <argument type="service" id="Shopware\Components\ShopwareReleaseStruct"/>
```

Comment:
this will not work in older SW versions, right?

Reply:
is already fixed...